src/Services/ExperianCredentialsService.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * This source file is proprietary and part of Rebilly.
+ *
+ * (c) Rebilly SRL
+ *     Rebilly Ltd.
+ *     Rebilly Inc.
+ *
+ * @see https://www.rebilly.com
+ */
+
+namespace Rebilly\Services;
+
+use JsonSerializable;
+use Rebilly\Entities\ExperianCredential;
+use Rebilly\Rest\Collection;
+use Rebilly\Rest\Service;
+
+final class ExperianCredentialsService extends Service
+{
+    /**
+     * @return ExperianCredential[]|Collection
+     */
+    public function search()
+    {
+        return $this->client()->get('credential-hashes/experian');
```

Comment:
Search without params? Seems strange

Reply:
I do not expect a lot of Experian credentials at the moment and as this feature is an urgent request I'd like to save some time on implementing filtering/sorting/pagination